pkg/model/appconfig.go

```diff
@@ -78,17 +80,72 @@ func (b *AppConfig) addToModel(model *BackstageModel, _ bsv1.Backstage) (bool, e
 }
 
 // implementation of RuntimeObject interface
-func (b *AppConfig) updateAndValidate(m *BackstageModel, _ bsv1.Backstage) error {
+func (b *AppConfig) updateAndValidate(m *BackstageModel, backstage bsv1.Backstage) error {
+	b.updateDefaultBaseUrls(m, backstage)
```

Comment:
I think a better way is to contribute baseUrls by calling it from route.updateAndValidate method, pretty much like files/envvars objects contribute mounts and envvars to deployment.
Have you tried this way?

Reply:
PTAL at [`ccd6ff3` (#892)](https://github.com/redhat-developer/rhdh-operator/pull/892/commits/ccd6ff3d2b0d197e057f2bef84ce317e3c29e94d) and [`9765a32` (#892)](https://github.com/redhat-developer/rhdh-operator/pull/892/commits/9765a3219fc287f07a13e34ea1994a6f8b24686a)